### DISCUSSION TITLE
Feature Request - Space Bar remap in congif

### DISCUSSION THREAD

**Discussion by DGmedic** [SethRobinson/UGT#42] [Ideas]
First off, amazing work! This is a really amazing utility with very thorough and clear directions. 

But...... I have one quality of life feature request.

I may have missed this option, but felt like I'd looked through dozens of times just be sure, but being able to remap the the "Continue After Translation" hotkey would be really helpful, especially by including gamepad control - such as being able to map it to L3 (Left Analog Button). Then there would be no need to ever touch my keyboard.

I lied, I have two requests - would it possible to allow users to enter a RBG/HEX code for translated text somewhere in fonts or config file?

**Comment by SethRobinson** [comment]
Hey, thanks for the feedback.

Ah, I see - there is an option "gamepad_button_to_scan_active_window|right joystick button" that allows you to set a button to start the capture, but I should also add a gamepad_button_to_cancel_scan to *cancel/close* the capture window also?

Currently it's hardcoded to close it, I guess I assumed since there aren't many buttons free in the UGT menu people would only want to configure the button to initiate it, but closing it works with B (on an xbox) or right stick button.  (where you push inward on the stick so it clicks)  I guess this would be a problem if you were using a SNES pad or something... a lot of UGT options would be missing.

[Example of me doing it here, looks like I'm pressing B on the xbox controller to close it, but right stick button also seems to work in the test I just did](https://twitter.com/rtsoft/status/1334016972237934592)

>>would it possible to allow users to enter a RBG/HEX code for translated text somewhere in fonts or config file?

Custom colors? Yeah, good idea.
